assets/styles/elements/_document.scss

```diff
@@ -5,14 +5,19 @@
 //
 // Simple page-level setup.
 //
-// 1. Set the default `font-size` and `line-height` for the entire project,
-//    sourced from our default variables.
+// 1. Include web fonts
 // 2. Ensure the page always fills at least the entire height of the viewport.
-//
+// 3. Set the default `font-size` and `line-height` for the entire project,
+//    sourced from our default variables.
+
+@include font-faces($font-faces, $font-dir); /* [1] */
+
 html {
     min-height: 100%; /* [2] */
+    line-height: $line-height;
+    font-family: ff("sans");
     color: $color;
-    line-height: $line-height; /* [1] */
+    line-height: $line-height; /* [3] */
```

Comment:
Hrm. I never noticed before but the reference numbers are using CSS comments but the referenced texts are using Sass comments.

I suggest we change `/* [N] */` to `// [N]` to not render in the processed (but not minified) CSS file.

Reply:
Updated